libraries/SITL/SIM_JSON.cpp

```diff
@@ -210,6 +210,33 @@ uint32_t JSON::parse_sensors(const char *json)
                 //printf("%s/%s = %i\n", key.section, key.key, *((unit8_t *)key.ptr));
                 break;
 
+            case DATA_FLOAT_ARRAY: {
+                // example: [18.0, 12.694079399108887]
+                if (*p++ != '[') {
+                    return false;
```

Comment:
```suggestion
                    return received_bitmask;
```
We do this everywhere else we early return

Reply:
done.